finagle-mysql/src/test/scala/com/twitter/finagle/mysql/integration/TypeTest.scala

```diff
@@ -26,58 +31,76 @@ class NumericTypeTest extends FunSuite with IntegrationClient {
       ) ENGINE=InnoDB DEFAULT CHARSET=utf8;"""))
 
     Await.ready(c.query(
-      """INSERT INTO `numeric` (`smallint`,
-        `tinyint`, `mediumint`, `int`,
-        `bigint`, `float`, `double`, `decimal`, `bit`)
-        VALUES (1, 2, 3, 4, 5, 1.61, 1.618, 1.61803398875, 1);"""))
+      """INSERT INTO `numeric` (
+        `tinyint`, `tinyint_unsigned`,
+        `smallint`, `smallint_unsigned`,
+        `mediumint`, `mediumint_unsigned`,
+        `int`, `int_unsigned`,
+        `bigint`, `bigint_unsigned`,
+        `float`, `double`, `decimal`, `bit`) VALUES (
+        127, 255,
+        32767, 63535,
+        8388607, 16777215,
+        2147483647, 4294967295,
+        9223372036854775807, 18446744073709551615,
+        1.61, 1.618, 1.61803398875, 1);"""))
+
+    val signedTextEncodedQuery = """SELECT `tinyint`, `smallint`, `mediumint`, `int`, `bigint`, `float`, `double`,`decimal`, `bit` FROM `numeric` """
+    runTest(c, signedTextEncodedQuery)(testRow)
+
+    // TODO Comment out after ignoreUnsigned = true
```

Comment:
You're going to need to make an way to configure the `Client` to operate in 'support unsigned' mode. This will entail making a `Stack.Param` and threading the boolean all the way through. Which will be a bit of a pain, but it is what it is. I'd remove the default argument for the Row constructors since you're always going to have to pass it up from a higher level anyway.

Stack.Params can be a little convoluted at times (all times, really), so look at the HTTP implementation for an example [here](https://github.com/twitter/finagle/blob/develop/finagle-http/src/main/scala/com/twitter/finagle/Http.scala#L78). The basic idea is to put typed 'params' (`Stack.Param[T]`) into a map where the stored items are typed based on the key (the `T`) and all params have a default.
It can then be configured using the pattern:
```scala
client.configured(SupportUnsignedInts(true))
  .whatever(..)
  .newService(..)
```